build_tools/update_environments_and_lock_files.py

```diff
@@ -80,7 +80,10 @@
 
 docstring_test_dependencies = ["sphinx", "numpydoc"]
 
-default_package_constraints = {}
+default_package_constraints = {
+    # XXX: Temporary work-around for pytest
```

Comment:
link to an issue for what's happening?

Reply:
Comment added with reference to Pytest PR. IMO this change makes sense, we need to tweak slightly our tests to take it into account.